index.php

```diff
@@ -54,7 +54,14 @@
     include(GLPI_ROOT . "/inc/includes.php");
     $_SESSION["glpicookietest"] = 'testcookie';
 
-   // For compatibility reason
+    //Try to detect GLPI agent calls
+    $rawdata = file_get_contents("php://input");
```

Comment:
Is there any header, like user-agent, that could be use to have a more specific filtering ?

Reply:
Nope, only glpi-agent send those kind of headers, and it's disabled right now